packages/strapi-hook-bookshelf/lib/index.js

```diff
@@ -1107,7 +1107,14 @@ module.exports = function(strapi) {
           result.key = `where.${key}`;
           result.value = {
             symbol: 'IN',
-            value,
+            value: _.isArray(value) ? value : [value],
```

Comment:
you can use _.castArray instead, it'll only cast the value to an array if it's not the case already

Reply:
Thanks, @kamalbennani I didn't know it.